airflow/config_templates/airflow_local_settings.py

```diff
@@ -60,10 +59,15 @@
     'version': 1,
     'disable_existing_loggers': False,
     'formatters': {
-        'airflow': {'format': LOG_FORMAT},
+        'airflow': {
+            'format': LOG_FORMAT,
+            'class': 'airflow.utils.log.timezone_aware.TimezoneAware',
```

Comment:
Let’s make this string a constant like `COLORED_FORMATTER_CLASS`.

Reply:
@uranusjr 

Like `COLORED_FORMATTER_CLASS`, I changed it to be user configurable. This is a bugfix, and backwards compatibility was adjusted in js, so a new class was designated as fallback.
Could you check it out?